server.go

```diff
@@ -950,9 +950,22 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 		IsOurAddress: cc.Wallet.IsOurAddress,
 		ContractBreach: func(chanPoint wire.OutPoint,
 			breachRet *lnwallet.BreachRetribution) error {
+
+			// processACK will handle the breachArbiter ACKing the
+			// event.
+			finalErr := make(chan error, 1)
+			processACK := func(brarErr error) {
+				if brarErr != nil {
```

Comment:
nit: in this commit this block could be simplified to be just `finalErr <-brarErr` and then could add the `if` in the next commit.